packages/aws-cdk-lib/aws-rds/test/cluster.test.ts

```diff
@@ -1713,7 +1713,66 @@ describe('cluster', () => {
     });
   });
 
-  describe('performance insights', () => {
+  describe('performance insights for cluster', () => {
+    test('cluster with all performance insights properties', () => {
```

Comment:
Can you add coverage for the scenario where `enablePerformanceInsights: true` and `performanceInsightRetention` / `performanceInsightEncryptionKey` are not specified?

Reply:
Yeah, I added: https://github.com/aws/aws-cdk/pull/31385/commits/217d94552e3f5953fe1588e7ea751385879c17ea